src/WPGraphQL.php

```diff
@@ -125,7 +125,7 @@ private function setup_constants() {
 
 		// Plugin version.
 		if ( ! defined( 'WPGRAPHQL_VERSION' ) ) {
-			define( 'WPGRAPHQL_VERSION', '1.3.4' );
+			define( 'WPGRAPHQL_VERSION', '1.3.5' );
```

Comment:
This version number down in this file seems like it's prone to disaster. Maybe another WP way to solve in a different issue ticket.

Reply:
Oh, I guess this is coming over from the 1.3.5 release branch and master into develop.

Reply:
Ah yes. . .I'm not the best at getting develop caught up with master after a release 🤦🏻‍♂️ 

Def something that can be automated with releases I think.